demos/HST/download_data_HST_template.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+
+import eureka.lib.mastDownload as md
+
+# Proposal/Program ID, can be string or int
+proposal_id = '13467'
+
+# List of one or more visit numbers
+visits = [60]
+# visits = np.arange(60,85)
+
+# Instrument name, can be upper or lower case.
+# Supported options include: WFC3, STIS, COS, or FGS.
+inst = 'WFC3'
+
+# Temporary download directory will be 'download_dir'/mastDownload/...
+download_dir = '.'
+
+# FITS file type (usually IMA, sometimes FLT)
+subgroup = 'IMA'
+
+# MAST API token for accessing data with exclusive access
+mast_token = None
+
+# Final destination of files after calling mast.consolidate
+final_dir = './HD209458/ima'
+
+# If data are public, no need to call md.login() or md.logout()
+md.login(mast_token)
+for vis in visits:
```

Comment:
I can't tell at a glance - are the different visits put into the same folder in the end or different folders?

Reply:
I can see now that it puts them into the same folder. I was going to suggest that they end up in different folders, but I know my phase curve and likely some other observations had to be broken into multiple visits, so perhaps it's fine to leave this as is.